doc/api/cli.md

```diff
@@ -975,6 +975,40 @@ surface on other platforms, but the performance impact may be severe.
 This flag is inherited from V8 and is subject to change upstream. It may
 disappear in a non-semver-major release.
 
+### `--env-file=config`
+
+> Stability: 1 - Experimental
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Loads environment variables from a file relative to the current directory.
+
+The format of the file should be one line per key-value pair of environment
+variable name and value separated by `=`:
+
+```text
+PORT=3000
+```
+
+Any text after a `#` is treated as a comment:
+
+```text
+# This is a comment
+PORT=3000 # This is also a comment
+```
+
+If the `NODE_OPTIONS` variable is set, it will be parsed to configure Node.js
+during startup. For example, if the following were saved into a `.env` file:
+
+```text
+NODE_OPTIONS="--enable-source-maps --inspect"
+```
+
+And run via `node --end-file .env`, the launched Node.js process will run
+with source maps enabled and the default port open.
```

Comment:
I just tested this and apparently `node: --inspector is not allowed in NODE_OPTIONS`. (Which we might want to change, but in a separate PR.)

```suggestion
NODE_OPTIONS="--enable-source-maps"
```

And run via `node --end-file .env`, the launched Node.js process will run
with source maps enabled.
```